src/main/java/edu/cornell/kfs/module/arl/CuArParameterConstants.java

```diff
@@ -0,0 +1,5 @@
+package edu.cornell.kfs.module.arl;
```

Comment:
Was the arl intentional? Or did you mean to put ar?

Reply:
good catch,  I shall change